cmd/kogito/command/error/kogitoapp_errors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package error
+
+import (
+	"fmt"
+)
+
+// KogitoAssetNotFound Custom error for handling build from file errors.
+type KogitoAssetNotFound struct {
```

Comment:
The struct name is misleading. From the name it looks like it marks just error related to missing asset, but in reality it is used for any asset build related error (for example when asset is not among supported types).
Can you please rename it, for example to something like KogitoAssetFileBuildError ?

Reply:
make sense, thanks,.